Cartfile

```diff
@@ -8,3 +8,4 @@ github "norio-nomura/Base32"
 github "DaveWoodCom/XCGLogger" "swift_1.2"   # Previously: ~> 1.8
 github "bignerdranch/Deferred" "master"
 github "CEWendel/SWTableViewCell"
+github "stephencelis/SQLite.swift" "master"
```

Comment:
We should figure out how to not have multiple SQLite implementations in the tree!